.github/workflows/test.yml

```diff
@@ -69,26 +71,76 @@ jobs:
             docker:
               - python/**
               - docker/**
+            docker_base:
+              - docker/base/**
```

Comment:
As a reminder, this second step here is checking to see if any changes have happened since the last push rather than in the pr